### DISCUSSION TITLE
Releases

### DISCUSSION THREAD

**Discussion by cthackers** [cthackers/adm-zip#524] [General]
@5saviahv Can you open discussions ? We can organize here new releases

**Comment by 5saviahv** [comment]
ok, yes I can. 

Currently there is #497 what is fixed already couple months and we should roll fix out ASAP.

I did add tests to check releases also in Windows to avoid similar issues. We still have little too few tests to say tests catch all typos 100% but still something.

**Reply by cthackers** [reply]
Exceptional job @5saviahv
I'm gonna do a npm push now, and then we should figure out what to add to release notes and do a github release and tag
